01_fib/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int) {
+
+	n1 := 0
+	n2 := 1
```

Comment:
You can fold this into one line: `n1, n2 := 0, 1`.

Reply:
Updated